src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,89 @@
+package duke;
+
+import duke.task.Task;
+
+import java.util.List;
+
+/**
+ * Handles all UI elements for duke.Duke.
+ */
+public class Ui {
+    /**
+     * Greets the user when Duke is started.
+     */
+    public static String greet() {
+        return "Hello, I'm Duke! I'm here to help you with ANYTHING";
```

Comment:
You might want to declare this string as a constant so that it is easier to edit or change in future